src/components/types.ts

```diff
@@ -166,6 +166,25 @@ export interface MultipartFormat extends DataFormat {
 
 export type MultipartData = MultipartFormat & Updates<MultipartFormat>
 
+export interface Answer {
+  label: string
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  value: any
```

Comment:
I wonder if we could narrow down the type for value. I think value won't store arrays or objects. String, number, bigint and boolean make sense to me. What do you think?